tests/foreman/cli/test_host.py

```diff
@@ -1787,11 +1793,23 @@ def test_positive_erratum_applicability(
     result = client.run(f'yum update -y --advisory {setup_custom_repo["security_errata"]}')
     assert result.status == 0
     client.subscription_manager_list_repos()
-    applicable_erratum = Host.errata_list({'host-id': host_info['id']})
-    applicable_erratum_ids = [
-        errata['erratum-id'] for errata in applicable_erratum if errata['installable'] == 'true'
-    ]
-    assert setup_custom_repo["security_errata"] not in applicable_erratum_ids
+    # verify that the applied erratum is not present in the list of installable errata
+    try:
+        applicable_erratum, _ = wait_for(
+            lambda: setup_custom_repo["security_errata"]
+            in [
+                errata['erratum-id']
+                for errata in Host.errata_list({'host-id': host_info['id']})
+                if errata['installable'] == 'true'
+            ],
+            handle_exception=True,
+            fail_condition=True,
+            timeout=300,
+            delay=5,
+        )
```

Comment:
The following is more readable to me. I spent a few minutes on it until I realized `fail_condition=True`. Do we achieve the same behavior if we would apply the patch?
```suggestion
        applicable_erratum, _ = wait_for(
            lambda: setup_custom_repo["security_errata"]
            not in [
                errata['erratum-id']
                for errata in Host.errata_list({'host-id': host_info['id']})
                if errata['installable'] == 'true'
            ],
            handle_exception=True,
            timeout=300,
            delay=5,
        )
```